client/my-sites/media-library/list.jsx

```diff
@@ -9,6 +9,7 @@ var ReactDom = require( 'react-dom' ),
 	findIndex = require( 'lodash/findIndex' );
 
 import { connect } from 'react-redux';
+import moment from 'moment';
```

Comment:
We could use our `localize` Higher Order Component which provides `moment` as a prop. Details here https://github.com/Automattic/i18n-calypso#localize